mon-pix/tests/acceptance/reset-password-test.js

```diff
@@ -41,6 +42,29 @@ describe('Acceptance | Reset Password Form', function() {
     // then
     expect(currentURL()).to.equal('/changer-mot-de-passe/brandone-reset-key');
     expect(find('.sign-form__body').textContent).to.contain('Votre mot de passe a été modifié avec succès');
+  });
+
+  it('should allow connected user when visit reset-password page', async () => {
```

Comment:
```suggestion
  it('should allow connected user to visit reset-password page', async () => {
```
J'avais pas fait attention sur le coup mais je pense que cette formulation est meilleure.